src/com/ms/silverking/text/ObjectDefParser2.java

```diff
@@ -102,7 +102,7 @@
     }
     
     public static String toClassAndDefString(Object o) {

```

Comment:
I think we shall remove the implicit dependency on `toString` for SKDef reflection, since:
- it's very easy for SK-Plugin Implementor to forget to override `toString` if not familiar with SKDef's convention
- the `toString` for a class may have other usage